ui/guest/src/react/GuestProxy.tsx

```diff
@@ -419,9 +419,9 @@ export function GuestProxy() {
           if (fieldType === 'image') {
             const tagName = updatedField.eq(index).prop('tagName').toLowerCase();
             if (tagName === 'img') {
-              updatedField.eq(index).attr('src', value);
```

Comment:
What was going on here?
Please retest this in other places too.

Reply:
updatedField is a query of all elements matching a modelId+fieldId, and it was only updating the src of one element (at `index`)

Reply:
The index needs to be **considered** for repeat-group-nested fields/item.